src/main/webapp/app/entities/programming-exercise/programming-exercise-detail.component.ts

```diff
@@ -45,21 +44,14 @@ export class ProgrammingExerciseDetailComponent implements OnInit {
             this.programmingExercise.solutionParticipation.programmingExercise = this.programmingExercise;
             this.programmingExercise.templateParticipation.programmingExercise = this.programmingExercise;
 
-            this.programmingExerciseParticipationService
-                .getLatestResultWithFeedback(this.programmingExercise.solutionParticipation.id)
-                .pipe(catchError(() => of(null)))
-                .subscribe((result: Result) => {
-                    this.programmingExercise.solutionParticipation.results = result ? [result] : [];
-                    this.loadingSolutionParticipationResults = false;
-                });
-
-            this.programmingExerciseParticipationService
-                .getLatestResultWithFeedback(this.programmingExercise.templateParticipation.id)
-                .pipe(catchError(() => of(null)))
-                .subscribe((result: Result) => {
-                    this.programmingExercise.templateParticipation.results = result ? [result] : [];
-                    this.loadingTemplateParticipationResults = false;
-                });
+            this.loadLatestResultWithFeedback(this.programmingExercise.solutionParticipation.id).subscribe(results => {
```

Comment:
This is just a refactoring I noticed @thilo-behnke started, but didn't complete. I am now just calling the `loadLatestResultWithFeedback` method that was already there